docs/advanced-topics/legacy/table--legtool-vs-ghatool.rst

```diff
@@ -0,0 +1,25 @@
++----------------------+-----------------------------+----------------------------------------+
```

Comment:
Or is this an "included" file, in which case it's never visible? Then it's fine.

Reply:
The extra dash is intentional, it's the naming schema I'm using for things that are only intended to be included in other files (with one exception: `snippet--installation.rst` does appear in the sidebar because it is crucial information I wanted to make as accessible as possible; when not imported, it will as you predicted will result in`--` in the URL).